### ISSUE TITLE
Alt Text for NTL Logo on home screen

### ISSUE THREAD

**Issue by weddleben** [Nashville-Public-Library/TL-mobile#56]
According to the [accessibility guidelines](https://www.w3.org/WAI/tutorials/images/decorative/), we should not use alt text for purely decorative images. However, a few days ago I talked to some blind/low vision folks who use screen readers and they said that every image should indeed have an `alt=""` attribute filled out. They want to know what is on the screen. Without it, they are curious/frustrated/confused. It should not say "NTL logo"; it should describe the logo.

So I think we should create a description for the NTL logo. Anyone want to volunteer for that? 

We can also use this description on the regular website and wherever else it's needed.

**Comment by weddleben** [comment]
@ddubBNA Fantastic! This is a great task for an LLM. The last two sentences are not relevant to the logo in the app. I love this, but is it too long?

**Comment by weddleben** [comment]
PERFECT

**Comment by weddleben** [comment]
This is done!

**Comment by travis-humbert** [comment]
This is a great description - and good work on finding the out that alt text should be descriptive like that Ben. Thank you all!

**Comment by travis-humbert** [comment]
That makes sense - in theory that'll be a good percentage of people using our app, so the more feedback the better.
